<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dl_project</class>
 <widget class="QDialog" name="dl_project">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="gbox_settings">
     <property name="title">
      <string>Project Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_conn_name">
        <property name="text">
         <string>Connection</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbx_conn_name"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pb_conn">
        <property name="toolTip">
         <string>Open Database Connections...</string>
        </property>
        <property name="text">
         <string>Choose...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_project_name">
        <property name="text">
         <string>Project Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="le_project_name">
        <property name="text">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbl_tolerance">
        <property name="text">
         <string>Tolerance</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="dsb_tolerance">
        <property name="specialValueText">
         <string/>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.001000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbl_logfile">
        <property name="text">
         <string>Log File</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="le_logfile"/>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="pb_logfile">
        <property name="toolTip">
         <string>Save Log File...</string>
        </property>
        <property name="text">
         <string>Save...</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lbl_crs">
        <property name="text">
         <string>CRS</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="cbx_crs">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="gbox_route_class">
     <property name="title">
      <string>Route Class</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="QLabel" name="txt_routeupdatetstz">
        <property name="text">
         <string>None</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_route_class_name"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_routeupdatetstz">
        <property name="text">
         <string>Last Update:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_route_class_name">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pb_add">
        <property name="toolTip">
         <string>Add Route Class layer to QGIS</string>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pb_remove">
        <property name="toolTip">
         <string>Remove Route Class layer from QGIS</string>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbx_conn_name</tabstop>
  <tabstop>pb_conn</tabstop>
  <tabstop>le_project_name</tabstop>
  <tabstop>dsb_tolerance</tabstop>
  <tabstop>le_logfile</tabstop>
  <tabstop>pb_logfile</tabstop>
  <tabstop>cbx_crs</tabstop>
  <tabstop>le_route_class_name</tabstop>
  <tabstop>pb_add</tabstop>
  <tabstop>pb_remove</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dl_project</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dl_project</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
